pkg/reconciler/subscription/subscription_test.go

```diff
@@ -0,0 +1,700 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Veroute.on 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package subscription
+
+import (
+	"encoding/json"
+	"fmt"
+	testing2 "github.com/knative/eventing/pkg/reconciler/testing"
```

Comment:
Can we call this import something more descriptive than `testing2`? I suggest `rectesting`

Reply:
as a followup?

Reply:
#1068